intermediate-python-testing.md

```diff
@@ -195,34 +196,51 @@ def database(request):
     pg_user = DB_OPTS.get("username")
     pg_db = DB_OPTS["database"]
     init_postgresql_database(pg_user, pg_host, pg_port, pg_db)
-
-    @request.addfinalizer
-    def drop_database():
-        # Last arg is postgres version
-        drop_postgresql_database(pg_user, pg_host, pg_port, pg_db, 9.6)
 ```
 
 DataMaders: If you need to add extensions to your test database, see [the `database` fixture in `dedupe-service`](https://github.com/datamade/dedupe-service/blob/master/tests/conftest.py).
 
-The above example includes two new and crucial concepts. Notice the scope keyword argument in our `fixture` decorator. Scope determines how long the context your fixture creates, sticks around. In `pytest`, there are three options:
+### Scope, or How to avoid confusing dependencies between your tests
+
+Notice the scope keyword argument in the `fixture` decorator above. Scope determines how long the context your fixture creates, sticks around. In `pytest`, there are three options:
```

Comment:
The second sentence feels a little clumsy. Maybe just: "Scope determine the lifespan of fixture's context."